src/components/WeatherCard/WeatherCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as S from './style';
+
+/* Import Components */
+import TransparentBox from '../TransparentBox';
+import Subtitle from '../Subtitle';
+import Icon from '../Icon';
+
+const WeatherCard = () => {
+  return (
+    <S.Card>
+      <TransparentBox color='lightgray' opacity='0.2'>
+        <Subtitle text='text1' />
+        <Subtitle text='text2' />
+        <Icon src='https://apidev.accuweather.com/developers/Media/Default/WeatherIcons/14-s.png' />
```

Comment:
add an option for choosing an icon from the app assets

Reply:
i fix it here : #14